### PR TITLE
[MM-31376] Do not subtract offset from accessories container

### PR DESCRIPTION
**Pull request by migbot**
#### Summary
Placed the autocomplate just above the post draft input at all times it is visible. For devices with smaller screens (iPhone SE and 12 mini), we reduce the max heights of both the autocomplete and the post draft input so that the autocomplete remains visible in the case of a multiline draft with file attachments.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-31376

#### Device Information
This PR was tested on:
* iPhone SE, 11, and 12 simulators
* Mi A3, Android 10

#### Screenshots
iPhone SE
SE

iPhone 11

iPhone 12 mini
12mini

Mi A3
Android

### CODE REVIEW

**Comment by amyblais** [comment]
Removing this from v1.38 for now; not a hard blocker and release tests should wrap up soon.

File: app/components/post_draft/post_draft.js [hunk 1]
```diff
@@ -121,6 +123,7 @@ export default class PostDraft extends PureComponent {
                 accessoriesContainerID={accessoriesContainerID}
                 ref={this.keyboardTracker}
                 scrollViewNativeID={scrollViewNativeID}
+                onLayout={onLayout}
```
Comment (enahum) [inline app/components/post_draft/post_draft.js:126]: this is not being used anywhere

**Review by enahum — CHANGES_REQUESTED** [review]
small change and it should be good to go

**Review by enahum — APPROVED** [review]

**Review by josephbaylon — APPROVED** [review]
LGTM. Autocomplete in single line and multi-line and attachment messages always appears on the top most edge of post draft and doesn't block post draft anymore. User is still able to select option from autocomplete. Tested on channel and thread.

Tested on iPhone X (14.3) and Pixel 3 XL (11) devices.

**Comment by enahum** [comment]
@migbot can you please cherry-pick this PR manually ?

**Pull request merged**